### ISSUE TITLE
Not working on Mac OS + Vivaldi

### ISSUE THREAD

**Issue by kklot** [jplattel/obsidian-clipper#37]
Clicking the icon shows this

![Screenshot 2022-02-06 at 17 18 39](https://user-images.githubusercontent.com/4459378/152690392-bc4166cd-5c3d-470d-a708-7ef1657a0b5f.png)
tick the allow box > open Obsidian > nothing happen.

Obsidian is running.

**Comment by jplattel** [comment]
Yes, Chrome needs permission. Check the box and retry while selecting a part of a page, it doesn't clip entire pages.

**Comment by ubuntudroid** [comment]
Regarding clipping pages: is this an issue specific to Vivaldi or specific to MacOS?

**Comment by hugodlmm** [comment]
Same here it doesn't work on Vivaldi W10
